article_store/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '3'
```

Comment:
This will need to be moved to the main `docker-compose.yml` (assume it's just here temporarily).

Reply:
Yer just for local dev for this PR, then I will move it across.